cfgov/v1/routing.py

```diff
@@ -54,7 +54,7 @@ def get_protected_url(context, page):
     if page:
```

Comment:
Yeah, it doesn't seem like we need it but we do return `#` if it's not live or doesn't exist, not sure if we still need that around. Agree we can deal with this in a separate PR.